USER_MANUAL.md

```diff
@@ -1064,29 +1056,19 @@ method_labels: ["aircraft","OGI_FU"]
 
 **Notes of caution:** Using a GWP of CH4 for a 20-year time period may dramatically change results but all options should be explored.
 
-#### &lt;sale_price_of_natural_gas&gt; WIP
+#### &lt;sale_price_of_natural_gas&gt; 
 
 **Data Type:** Numeric
 
 **Default input:** 3.0
```

Comment:
Is this value still correct - We went from mcf to MBTU

Reply:
Yes.